redis/client.py

```diff
@@ -268,6 +267,13 @@ def __init__(
                 "redis_connect_func": redis_connect_func,
                 "credential_provider": credential_provider,
                 "protocol": protocol,
+                "cache_enable": cache_enable,
```

Comment:
@sjpotter @vladvildanov @ofekshenawa Can you all align on these, with whatever is right _collectively_? Then, we can get this into the specification.